tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/ScalaJSClassEmitter.scala

```diff
@@ -880,6 +885,27 @@ private[emitter] final class ScalaJSClassEmitter(
     )
   }
 
+  def genTopLevelExportDef(cd: LinkedClass, tree: TopLevelExportDef)(
+      implicit globalKnowledge: GlobalKnowledge): js.Tree = {
+    import TreeDSL._
+
+    val MethodDef(_, StringLiteral(fullName), args, resultType, Some(body)) =
+        tree.member
```

Comment:
Maybe use `true` instead of `_`. Trying to emit a non-static top-level method should not silently succeed.